dev/com.ibm.ws.jsp.2.3_fat/test-applications/TestEDR.war/resources/index.jsp

```diff
@@ -16,6 +16,8 @@
         <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
         <title>TestEDR</title>
         <%@ taglib prefix="test" uri="/WEB-INF/tags/sampleTag.tld" %>
+        <!-- The following taglib is purposely a tld that does not exist.  For issue 18941 that fixes NPE. -->
+        <%@ taglib prefix="nf" uri="/WEB-INF/pluto/notplanet.tld" %>
```

Comment:
I see you're adding a tld that does not exist but where are we checking for the NPE in the actual test client  code?

Reply:
Also remember we're in 2022 now so if you think this is a significant enough change to the jsp please add the 2022 to the copyright date range.

Reply:
> I see you're adding a tld that does not exist but where are we checking for the NPE in the actual test client code?

I see what you're doing here now the product code is fixed so we should never see an NPE there isn't any need to update the test  client! Thanks Evan for the clarification.

Reply:
date fixed